Remove Luxi Mono font.

The Luxi fonts are licensed under the "Bigelow & Holmes Inc and URW++ GmbH
Luxi font license" (included as Licenses/LUXI License.txt) which is not
compliant with Fedora's Licensing Guidelines.

Fortunately, gargoyle has a Jamrules flag we set to tell it not to depend on the
fonts that usually come bundled with it. Instead, this package depends on
packages linux-libertine-fonts and liberation-mono-fonts, which gargoyle uses
instead.

--- a/src/Jamrules
+++ b/src/Jamrules
@@ -4,7 +4,7 @@
 
 BUILD ?= RELEASE ;
 USESDL = yes ;
-BUNDLEFONTS = yes ;
+BUNDLEFONTS = no ;
 
 CC      = gcc ;
 C++     = g++ ;
--- a/src/License.txt
+++ b/src/License.txt
@@ -26,8 +26,6 @@ SCARE is copyright (c) Simon Baldwin. It is released under the terms of the GNU
 
 TADS is copyright (c) Michael J. Roberts. It is released under the terms of the GNU General Public License.
 
-The Luxi fonts are copyright (c) Bigelow & Holmes Inc. They are released under the terms of the Bigelow & Holmes Inc and URW++ GmbH Luxi font license.
-
 The Bitstream Charter fonts are designed by Matthew Carter and copyright (c) Bistream Inc., Cambridge, MA. You are hereby granted permission under all Bitstream propriety rights to use, copy, modify, sublicense, sell, and redistribute the 4 Bitstream Charter (r) Type 1 outline fonts and the 4 Courier Type 1 outline fonts for any purpose and without restriction; provided, that this notice is left intact on all copies of such fonts and that Bitstream's trademark is acknowledged as shown below on all unmodified copies of the 4 Charter Type 1 fonts. BITSTREAM CHARTER is a registered trademark of Bitstream Inc.
 
 The Linux Libertine fonts are copyright (c) Philipp H. Poll. They are released under the terms of the SIL Open Font License.

